dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoiceBehaviorConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Represents function choice behavior configuration produced by a <see cref="FunctionChoiceBehavior" />.
+/// </summary>
+public sealed class FunctionChoiceBehaviorConfiguration
```

Comment:
If this is a strategy, suggest using `Strategy`, gives a better understanding.

```suggestion
public sealed class FunctionChoiceBehaviorStrategy
```

Reply:
It's not a strategy in its current form; it's just a configuration used by AI connectors to define the way functions are chosen by the LLM and various aspects of their invocation by AI connectors.